### PR TITLE
Infra/add tf and helm

### PR DESCRIPTION
**Pull request by karolmartyna**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -69,4 +69,4 @@ services:
 volumes:
   config:
   repo:
-  db-data:
+  db-data:
```
Comment (szewczykmira) [inline docker-compose.yml:72]: Does your linter removed new line or was it deliberate action?

File: infra/README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+# WIP
+DRAFT OF README
```
Comment (szewczykmira) [inline infra/README.md:2]: Even if this readme will be imporved over the time then IMO you can ommit those "WIP" notes :)

File: infra/README.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+# Setup AWS infra
+Requirements:
+```
+aws-cli >=v2.0.29
```
Comment (szewczykmira) [inline infra/README.md:7]: But I think that providing links to the download page would simplify the installation process. My first thought was to type `pip install aws-cli` and I'm pretty sure that's not the right way to install them ;)

File: infra/README.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+terraform >=0.12.20
+helm >=v3.2.4
+```
+`aws configure`
```
Comment (szewczykmira) [inline infra/README.md:11]: Those are still requirements or commands to perform?

File: docker-compose.yml (hunk 1 above)
Comment (karolmartyna) [reply docker-compose.yml:72]: I've added a temporary new volume for test and it was probably a reason.

**Comment by karolmartyna** [comment]
I've rebuilt README with more information about the setup - including #25  as well.

**Review by szewczykmira — APPROVED** [review]

**Pull request merged**